MAVProxy/mavproxy.py

```diff
@@ -1364,7 +1360,7 @@ def quit_handler(signum = None, frame = None):
         start_script = os.path.join(os.environ['HOME'], ".mavinit.scr")
         start_scripts.append(start_script)
     if 'LOCALAPPDATA' in os.environ and not opts.setup:
-        start_script = os.path.join(os.environ['LOCALAPPDATA'], "MAVProxy", "mavinit.scr")
+        start_script = os.path.join(os.environ['LOCALAPPDATA'], ".mavproxy", "mavinit.scr")
```

Comment:
Is it appropriate to use a dot-directory in `LOCALAPPDATA`?

I see you've marked it as "consolidate" - but it doesn't seem right to use a dot-directory in there....

Reply:
The ``.mavproxy`` is used to store the parameter definitions from ``param download``. I was consolidating touse that folder, rather than have multiple place that MAVProxy stores data in the user home folder.